functions/src/modules/SQR.functions.ts

```diff
@@ -20,52 +20,58 @@ export enum ISoundQualityReportSheet {
   Submissions = 'Submissions',
 }
 
-/////////////////////////////////////////////////
-//          OnNewAllotment (DB create and update Trigger)
-//      1. Mark the files in the database --> { status: "Given" }
-//              Function --> updateFilesOnNewAllotment
-//
-//      2. Send an email to the assignee to notify them of the new allotments
-//              Function --> sendEmailOnNewAllotment
-/////////////////////////////////////////////////
-export const updateFilesOnNewAllotment = functions.database
-  .ref('/sqr/allotments/{allotment_id}')
-  .onCreate((snapshot, context) => {
-    const allotment = snapshot.val();
-    let newDocKey = snapshot.key;
-
-    // loop through the FILES array in the NEW ALLOTMENT object
-    // and update their corresponding file objects
-    allotment.files.forEach(async file => {
-      let sqrRef = db.ref(
-        `/files/${allotment.list}/${file}/soundQualityReporting`
-      );
 
-      let sqrError = await sqrRef.update({
-        status: 'Given',
-        assignee: allotment.assignee,
-        timestampGiven: Math.round(new Date().getTime() / 1000),
-        timestampDone: null,
-      });
+/**
+ * Process registration of webform data
+ * 1. Registers user by transferring data into /users/ path
+ * 2. Check if users already exists
+ * 
+ * @function processRegistration()
+ */
+export const processRegistration = functions.database
+.ref('/webforms/registration/{registration_id}')
+.onCreate(async (snapshot, context) => {
+  const user = snapshot.val();
```

Comment:
To avoid confusion I would suggest
```suggestion
  const webform = snapshot.val();
```